app/services/i18n/i18n.ts

```diff
@@ -187,6 +189,10 @@ export class I18nService extends PersistentStatefulService<II18nState> implement
     return dictionary;
   }
 
+  private localeIsSupported(locale: string) {
+    return WHITE_LIST.includes(locale) && fs.existsSync(`${this.getI18nPath()}/${locale}`)
```

Comment:
if this is executed synchronously do we need to add an await when it's invoked or does fs handle that for us?

Reply:
`fs.existsSync` is a synchronous method, Adding `await` before synchronous method makes this method asynchronous